app/src/main/java/com/gogiyumandroid/GoogleMap.kt

```diff
@@ -0,0 +1,126 @@
+package com.gogiyumandroid
```

Comment:
You can keep this file now. But later don't add class files which is not used in this commit.